event_attribution_reporting_views.md

```diff
@@ -1,4 +1,4 @@
-# Event Attribution Reporting API Explainer
+# Attribution Reporting API for View-Through
```

Comment:
@johnivdel Is this OK?

Reply:
We could consider doing "Attribution Reporting for View-Through Measurement" and "Attribution Reporting for Click-Through Measurement" which align slightly better with the other explainer names.

But these seem fine as well.

Reply:
Used your suggestion. Thank you!